GitSearch/Application/SearchResult/SearchResultViewController.swift

```diff
@@ -16,7 +16,8 @@ class SearchResultViewController: UICollectionViewController, UICollectionViewDe
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        collectionView?.backgroundColor = UIColor.white
+        collectionView?.backgroundColor = UIColor(red: 235/255, green: 235/255, blue: 235/255, alpha: 1.0)
```

Comment:
There is a nice feature in Xcode called [#colorLiteral](https://medium.com/ios-os-x-development/be-literal-76e9b4389eda)

Reply:
Thank, I will think about it - for the first time it looks weird for me